### PR TITLE
Updating GP

### PR DESCRIPTION
**Pull request by sofianehaddad**
Add new features to `Kriging` in order to handle easily normalization, optimization, hmat/lapack...

### CODE REVIEW

File: otsklearn/otsklearn.py [hunk 1]
```diff
@@ -144,21 +144,31 @@ def predict(self, X):
...
+                 n_iter_opt = 100, normalize_data = True, linalg_meth = "LAPACK"):
         """Kriging estimator.
 
         Parameters
         ----------
-        kernel : str
+        kernel : str or ~openturns.CovarianceModel
             Covariance model type
-        basis : str
+        basis : str or ~openturns.CovarianceModel
```
Comment (jschueller) [inline otsklearn/otsklearn.py]:
```
:py:class:`openturns.Basis`
```
